lib/measured/rails/active_record.rb

```diff
@@ -45,8 +45,8 @@ def measured(measured_class, *fields)
             precision = self.column_for_attribute(value_field_name).precision
             scale = self.column_for_attribute(value_field_name).scale
             rounded_to_scale_value = incoming.value.round(scale)
```

Comment:
This scares me, I have a feeling that ActiveRecord should already round so this runs the risk of rounding too much or double-rounding losing precision.